Update hub pages

jgagne opened the pull request:
## What this PR does:

- Add "A Decade of Responsive Web Design" webinar content to Web Development and UX Design collection landing pages
- Update webinar related alt text descriptions to include guest names, fix typo, remove company affiliation, and remove periods
- Update webinar section description to include plural form of "conversation" to read: "Insightful conversations with trailblazers in the web community."
- Update `/webinars/right-sizing-your-rapid-prototypes` webinar poster artwork filename to match URL
- Update Remove Work featured artwork filename
- Update hub page featured artwork performance with image async decoding hint

- - -

**Web Development Collection: A Decade of RWD Update**

![decade-of-rwd-featured-webinar-promo](https://user-images.githubusercontent.com/5142085/84058641-8711ef00-a987-11ea-8196-d398e0a53c6e.png)

![decade-of-rwd-webinar-section](https://user-images.githubusercontent.com/5142085/84060923-0228d480-a98b-11ea-9a88-9f41e7f08fe0.png)

**UX Design Collection: A Decade of RWD Update**

![ux-decade-of-rwd-featured-webinar-promo](https://user-images.githubusercontent.com/5142085/84070792-b1b97300-a99a-11ea-8bf1-a79583af1c58.png)

jgagne committed:
Add webinar poster artwork

- Add webinar poster artwork for “A Decade of Responsive Web Design”

jgagne committed:
Update webinar poster artwork filename

Update webinar poster artwork filename to match webinar URL; right-sizing-your-rapid-prototypes

jgagne committed:
Update webinar featured artwork

- Update webinar UI element in artwork

jgagne committed:
Update featured artwork performance

- Add image async decoding hint to featured artwork

jgagne committed:
Update remote work webinar artwork

- Update filename
- Update alt text

jgagne committed:
Add web development webinar

- Add “A Decade of Responsive Web Design” webinar
- Update featured webinar with current promo
- Add image decoding hint to featured artwork

jgagne committed:
Update webinar alt text

- Update webinar alt text description with both guests
- Add period to alt text description
- Remove company affiliation

jgagne committed:
Update webinars section description

- Update “conversation” to “conversations”

jgagne committed:
Update webinar artwork path

- Update webinar poster artwork filename to match webinar URL; right-sizing-your-rapid-prototypes

jgagne committed:
Update webinar artwork crop

- Remove “poster-mask-crop” class now that the artwork is the correct dimensions

andrewpmiller reviewed (commented):
@jgagne — what are your thoughts on adding the Decade of RWD to the UX landing page webinars as well? (And/or featured?) — just a thought I had so I figured I'd ask you.

jgagne commented:
@andrewpmiller — I'm cool with adding it the UX landing page; it's fresh and relevant content. I'll work on an update.

jgagne committed:
Update featured course alt text

- Modern Web Design

jgagne committed:
Add recent webinar to UX Design collection

- Add “A Decade of Responsive Web Design” webinar to UX Design collection featured and webinar sections
